homeassistant/components/samsungtv/media_player.py

```diff
@@ -158,12 +162,21 @@ def update(self) -> None:
         else:
             self._attr_state = STATE_ON if self._bridge.is_on() else STATE_OFF
 
-    def send_key(self, key: str) -> None:
+        if self._attr_state == STATE_ON and self._app_list is None:
+            self._app_list = {}  # Ensure that we don't update it twice in parallel
+            self.hass.async_add_job(self._update_app_list)
```

Comment:
We are not allowed to use the async api from sync context. Probably use `hass.add_job` if we want to schedule this call. Otherwise we can just call it directly.